app/config.py

```diff
@@ -47,6 +47,8 @@ class Config(object):
 
     DEFAULT_SERVICE_LIMIT = 50
 
+    GLOBAL_SERVICE_MESSAGE_LIMIT = 5000
```

Comment:
Ah, I see that this is necessary for calculating the remaining amount, not just showing the limit. Let's open an issue for `notifications-utils` to move the limit there, and then we can reference that in both applications without needing to maintain them in parallel.

Reply:
👍 @stvnrlly!

@A-Shumway42 and I discussed the same thing when pairing through this, and I was suspecting that having it be managed and accessible in `notifications-utils` would make the most sense rather than having it hardcoded in multiple spots.

I agree:  finish this up and merge it, then create a separate issue for that work and follow up in here again once `notification-utils` has full support for it.